apps/web/src/actions/rewards/fetchPendingRewardClaimsAction.ts

```diff
@@ -0,0 +1,17 @@
+'use server'
+
+import { findAllRewardClaimsPendingToValidate } from '@latitude-data/core/data-access'
+import { UnauthorizedError } from '@latitude-data/core/lib/errors'
+
+import { authProcedure } from '../procedures'
+
+export const fetchPendingRewardClaimsAction = authProcedure
+  .createServerAction()
+  .handler(async ({ ctx }) => {
+    if (!ctx.user.admin) {
```

Comment:
Do a procedure `adminProcedure`. Is almost certain that we'll use it in the future in other actions